headers/types/ground_mode/live_actor.h

```diff
@@ -0,0 +1,98 @@
+#ifndef HEADERS_TYPES_GROUND_MODE_LIVE_ACTOR_H_
```

Comment:
This doesn't feel substantial enough to split out into a separate file; there's only two structs here. Can we just put these definitions directly in `ground_mode.h`? That file isn't super long anyway.